src/IRremoteESP8266.h

```diff
@@ -333,9 +336,10 @@ enum decode_type_t {
   GOODWEATHER,
   INAX,
   DAIKIN160,  // 65
+  DAIKIN176,
```

Comment:
`NEOCLIMA` needs to come before `DAIKIN176`
e.g. NEOCLIMA is already 66, DAIKIN176 will be 67.

See comment in the code:
```
/*
 * Always add to the end of the list and should never remove entries
 * or change order. Projects may save the type number for later usage
 * so numbering should always stay the same.
 */
```